planner:
  sot_type: "SoTStatic"
  tasks:
    - name: "Base Pos 1"
      planner_type: "BaseSingleWaypoint"
      target_pos: [2.0, 0.]
      tracking_err_tol: 0.25
#    - name: "EE Pos 1"
#      planner_type: "EESimplePlanner"
#      target_pos: [1.5, -0.5, 1.]
#      hold_period: 3.
#      tracking_err_tol: 0.02

controller:
  mpc_rate: 10 #hz



include:
  - package: "mmseq_run"
    path: "config/robot/thing.yaml"
  - package: "mmseq_run"
    path: "config/controller/HTMPC.yaml"
  - package: "mmseq_run"
    path: "config/sim/simulation.yaml"

logging:
  log_dir: "/home/tracy/Documents/PhD/data/debug"
  log_level: 4 # 0 - not set, 10 - debug, 20 - info
